docs/.gitignore

```diff
@@ -5,3 +5,4 @@ yarn-error.log
 /cypress/screenshots/
 /__sapper__/
 .env
+.svx
```

Comment:
What is that for?

Reply:
It was a folder that got generated when I ran `yarn docs:export`

Reply:
Can't reproduce. It shouldn't generate any folders

Reply:
Maybe it's a Windows thing, idk

Reply:
Doesn't hurt, anyway